templates/layout.html

```diff
@@ -157,6 +157,16 @@
             </div>
         </nav>
         {% endblock %}
+        {%- if announcements %}
+            <div class="container">
+                {%- for announcement in announcements %}
+                    <div class="well alert alert-danger" style="padding:0.5em 1em;">
+                        <h1>{{ announcement.title }} <small>{{ announcement.created }}</small></h1>
```

Comment:
Can we also format the creation time here? I think it's okay to keep the hour + minutes here, but we don't really need seconds and millionth of a second.

Reply:
Done.